zeebe/backup/src/main/java/io/camunda/zeebe/backup/api/BackupManager.java

```diff
@@ -36,13 +36,17 @@ public interface BackupManager {
    */
   ActorFuture<BackupStatus> getBackupStatus(long checkpointId);
 
+  default ActorFuture<Collection<BackupStatus>> listBackups() {
+    return listBackups(null);
```

Comment:
❓ No Wildcard?

Reply:
Outdated, this code was overwritten later on.